packages/core/test/history.test.ts

```diff
@@ -232,66 +232,64 @@ describe('parallel history states', () => {
     }
   });
 
-  describe('history', () => {
-    // on.first -> on.second.A
-    const stateABKL = historyMachine.transition(
-      historyMachine.initialState,
-      'SWITCH'
-    );
-    // INNER_A twice
-    const stateACDKL = historyMachine.transition(stateABKL, 'INNER_A');
-    const stateACEKL = historyMachine.transition(stateACDKL, 'INNER_A');
+  // on.first -> on.second.A
```

Comment:
None of these tests were changed; they were just unnecessarily nested in a `describe` block, so I flattened them.